source/_usecases/format-curreny-as-per-the-locale.md

```diff
@@ -0,0 +1,32 @@
+---
+extends: _layouts.usecase
+date: 2018-03-04
+link: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString
+category: number
+---
+
+
+```javascript
+function formatCurrency(value, locale, currency) {
```

Comment:
please simply show how to use it without a function, because we like to show the "step by step" behind our approach, rather than ready made functions
So simply get rid of the function and keep then use `amount.toLocaleString..`

Reply:
Good idea, rather than a ready-made solution step by step approach is better.